denis/utilities.py

```diff
@@ -102,6 +103,34 @@ def check_corrupt_or_missing(repo, tag, username_to_subs):
     return msg
 
 
+def check_signed_off_by(repo, tag):
+    [_, _, user] = tag.split('_')
+    hostname = os.getenv("HOSTNAME")
+
+    msg = 'signed off by check'
+    msg += '\n'
+    msg += '-------------------'
+    msg += '\n\n'
+
+    commits = repo.git.execute(['git', 'rev-list', tag]).split('\n')
```

Comment:
I can do you even one better than shuffling where to call the `reversed` around in the python. Just pass `--reverse` to the git rev-list command to get it to give you them in the right order